### PR TITLE
Switch to setup-node@v2 for built-in npm caching

### PR DESCRIPTION
**Pull request by whiten**

### CODE REVIEW

**Comment by arv** [comment]
It is a lot cleaner so LGTM

It does not seem to impact the time much thogh

**Comment by arv** [comment]
Are you still working on this one?

File: .github/workflows/js.yml [hunk 1]
```diff
@@ -35,12 +39,14 @@ jobs:
...
+      PLAYWRIGHT_BROWSERS_PATH: 0
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
           node-version: 14.x
           cache: 'npm'
       - run: npm install
-      - run: npx playwright install-deps
+      - run: npx playwright install-deps chromium
```
Comment (arv) [inline .github/workflows/js.yml]: That does not seem right. We run the tests on Firefox and WebKit too

**Review by arv — CHANGES_REQUESTED** [review]

File: .github/workflows/js.yml (hunk 1 above)
Comment (whiten) [reply .github/workflows/js.yml]: Yeah, I mixed this up with perf tests, where we default to only chromium.

**Pull request merged**